src/Domain/Creators/SphereTimeDependentMaps.hpp

```diff
@@ -223,28 +277,32 @@ struct TimeDependentMapOptions {
       bool include_distorted_map) const;
 
   /*!
-   * \brief This will construct the map from `Frame::Grid` to `Frame::Inertial`.
+   * \brief This will construct the map from `Frame::Grid` to
+   * `Frame::Inertial`.
    *
-   * If the argument `include_distorted_map` is true, then this map will have a
-   * `Shape` map (with a size function of time). If it is false, then there will
-   * only be an identity map.
+   * If the argument `include_distorted_map` is true, then this map will
+   * have a `Shape` map (with a size function of time). If it is false, then
+   * there will only be an identity map.
```

Comment:
This is no longer true. There will be rotation, expansion, and translation maps now. Same for the other `map` functions.